run_uh.py

```diff
@@ -50,19 +50,15 @@
 # this will break any run_uh imports from other locations (e.g. _get_version())
 
 def exit_with_error(title, message):
-	if title + '.' != message:
-		print(title)
-	print(message)
-
+	print(title + '\n' + message);
 	try:
 		import Tkinter
 		import tkMessageBox
 		window = Tkinter.Tk()
 		window.wm_withdraw()
 		tkMessageBox.showerror(title, message)
 	except ImportError:
-		# tkinter may be missing
-		pass
+		print("Tkinter module missing. Install Tkinter for Python2.7 to view pop-up error dialogs");
```

Comment:
But I leave that up to you, but I'd like to see at least a newline between error and tkinter notice.